packages/block-library/src/file/edit.native.js

```diff
@@ -138,34 +262,87 @@ export default class FileEdit extends Component {
 				}
 				onFinishMediaUploadWithFailure={ () => {} }
 				onMediaUploadStateReset={ this.mediaUploadStateReset }
-				renderContent={ ( { isUploadFailed, retryMessage } ) => {
+				renderContent={ ( {
+					isUploadInProgress,
+					isUploadFailed,
+					retryMessage,
+				} ) => {
 					if ( isUploadFailed ) {
 						return this.getErrorComponent( retryMessage );
 					}
 
 					return (
 						<View>
+							{ isUploadInProgress ||
+								this.getToolbarEditButton( openMediaOptions ) }
+							{ getMediaOptions() }
+							{ this.getInspectorControls( attributes ) }
 							<RichText
 								__unstableMobileNoFocusOnMount
```

Comment:
@etoledom just a thought here. Do you think we should disable the user's ability to edit the filename while the upload is taking place? I was just doing an upload and when I realized I could still interact with it, it felt like something that should be done when we are sure the file upload is successful. Looking forward to hearing your thoughts on this.

Reply:
Good catch!
I just tested on the Web version and we are able to edit the file name also there. Since it's no more than an attribute change I don't think it's a big deal (unless there's any related bug/UX issue with this).

I personally tend to prefer copying what's in Web, but not hard feelings.
@iamthomasbishop - any thought about this?

Reply:
What we should definitely block is the Copy URL button.
Done in https://github.com/WordPress/gutenberg/pull/26360/commits/e777c4b8ed77cc04a647d92effa5c5f1d114720c

Reply:
![disable_button](https://user-images.githubusercontent.com/9772967/99256210-59be4c80-2815-11eb-9ac6-15ec9208ab6b.gif)

cc @iamthomasbishop

Reply:
> What we should definitely block is the Copy URL button.

This sounds like a good idea 😄

Reply:
> we should definitely block is the Copy URL button.

Can we set the button to a disabled state? Not sure if we already have a style for that. If not, we can probably set the opacity to 25%, which will end up being our standard `quaternary`/`disabled` style when our new color system is fully-implemented.

Reply:
Thank you @iamthomasbishop ! Done in https://github.com/WordPress/gutenberg/pull/26360/commits/262475dec54632d810696e8ba7937047039e52b7

I also changed the Download button to 25% opacity when disabled.

![link_button](https://user-images.githubusercontent.com/9772967/99361927-1c100100-28b3-11eb-9e1b-e2505763cd85.gif)

Reply:
@etoledom that's definitely better. One more tiny thing: can we also change the text-color (in the `disabled` state) to white?